### DISCUSSION TITLE
https://5kdk.github.io/blog/2023/10/18/suspense-for-data-fetching

### DISCUSSION THREAD

**Discussion by giscus** [5kdk/5kdk.github.io#5] [General]
# https://5kdk.github.io/blog/2023/10/18/suspense-for-data-fetching

비동기처리에서의 Suspense와 ErrorBoundary 활용법.

https://5kdk.github.io/blog/2023/10/18/suspense-for-data-fetching

**Comment by sqsung** [comment]
좋은 글 공유해주셔서 감사합니다! 잘 읽고 갑니다.

**Comment by ryu9663** [comment]
감사합니다
